lib-utilities/config/odimra_config.json

```diff
@@ -1,23 +1,23 @@
 {
-	"RootServiceUUID": "",
-	"LocalhostFQDN": "",
-	"SearchAndFilterSchemaPath": "",
-	"RegistryStorePath": "",
+	"RootServiceUUID": "ac2f7880-9d8a-11ed-a8fc-0242ac120002",
+	"LocalhostFQDN": "deviceManager",
+	"SearchAndFilterSchemaPath": "/etc/deviceManager/configs/schema.json",
```

Comment:
Can't we use schema.json file located in /lib-utilities/config?

Reply:
Configs and certs should be moved to /etc/deviceManager/ directory.